src/utils/read-pacts.ts

```diff
@@ -8,13 +8,26 @@ export function readPacts(dir: string): Pact[] {
         throw new Error(`Pact directory ${dir} is empty.`);
     }
 
-    return fileNames.map((fileName) => {
+    const pacts = fileNames.map((fileName) => {
         const filePath = path.join(dir, fileName);
-        return JSON.parse(fs.readFileSync(filePath, 'utf8'));
+        return JSON.parse(fs.readFileSync(filePath, 'utf8')) as Pact;
     });
+
+    const pactWithEmptyInteractions = pacts.find(
+        (pact) => pact.interactions.length === 0
+    );
+
+    if (pactWithEmptyInteractions) {
+        throw new Error(
+            `Pact interactions for provider ${pactWithEmptyInteractions.provider?.name} are empty.`
```

Comment:
I think that putting `filePath` in the error message would be more helpful for debugging than provider name:

```
No interactions found in pact file /path/to/pact.json.
```

Reply:
Instead of provider name rather than additionally?

Reply:
Both would be even better!